src/bin/wasmer.rs

```diff
@@ -492,6 +492,20 @@ fn execute_wasm(options: &Run) -> Result<(), String> {
         None
     };
 
+    // Update backend when a backend flag is `auto` or a default value.
```

Comment:
> or a default value

Is it still true? By reading the code, I assume it's when the `--backend` option is set to `auto` only?

Reply:
I overlooked the Backend::default(). I updated the return value from `Cranelift` to `Auto` in Backend::default(), so this comment should be true.